recipe/meta.yaml

```diff
@@ -50,7 +50,7 @@ about:
     in parameter space. With just a few lines of code you can evaluate
     functions on a computing cluster, live-plot the data as it returns,
     and fine-tune the adaptive sampling algorithm.
-  doc_url: https://nbviewer.jupyter.org/github/python-adaptive/adaptive/blob/master/learner.ipynb
+  doc_url: http://adaptive.readthedocs.org
   dev_url: https://github.com/python-adaptive/adaptive/
```

Comment:
Should we also not change this as well, in view of https://github.com/python-adaptive/adaptive/issues/5?

Reply:
Both @jbweston and I hope to move the development to GitHub some day.

Reply:
No objections from my side, but right now it's not there...